src/Tasks/Microsoft.NET.Build.Tasks/IPackageResolver.cs

```diff
@@ -8,5 +8,6 @@ namespace Microsoft.NET.Build.Tasks
     public interface IPackageResolver
     {
         string GetPackageDirectory(string packageId, NuGetVersion version);
+        string GetPackageDirectory(string packageId, NuGetVersion version, out string packageRoot);
```

Comment:
@srtivatsn, what's going to be our policy on breaking changes to API surface area in Microsoft.NET.Build.Tasks. We have stuff that's public for unit testability but not intended to be called by customers. Should we lock it down with IVTA before RTM?